src/client/modell/UtbetalingshistorikkElement.ts

```diff
@@ -28,112 +27,50 @@ export const utbetalingshistorikkelement = (
-                type: Periodetype.VEDTAKSPERIODE,
-                tilstand: utbetaling?.status ?? Utbetalingstatus.INGEN_UTBETALING,
-                utbetalingstidslinje: utbetaling ? utbetalingstidslinje(utbetaling, it.fom, it.tom) : [],
-                sykdomstidslinje: it.sykdomstidslinje ?? [],
-                organisasjonsnummer: organisasjonsnummer,
-            };
-        });
-};
+export const erRevurderingsperiode = (periode: Tidslinjeperiode) => periode.type === Periodetype.REVURDERING;
 
-const revurderingsperioder = (
-    beregningsId: string,
-    beregnetTidslinje: Sykdomsdag[],
-    organisasjonsnummer: string,
-    utbetaling?: UtbetalingshistorikkUtbetaling2
-): Tidslinjeperiode[] => {
-    if (!utbetaling) return [];
-    const perioder = new PeriodeBuilder().build(
-        beregnetTidslinje.filter((it) => it.dato.isSameOrAfter(utbetaling?.utbetalingstidslinje[0].dato))
-    );
-    return perioder.map((it) => {
-        return {
-            id: nanoid(),
-            beregningId: beregningsId,
-            fom: it.fom,
-            tom: it.tom,
-            tilstand: utbetaling.status ?? Utbetalingstatus.INGEN_UTBETALING,
-            utbetalingstidslinje: utbetaling ? utbetalingstidslinje(utbetaling, it.fom, it.tom) : [],
-            sykdomstidslinje: sykdomstidslinje(beregnetTidslinje, it.fom, it.tom),
-            type: Periodetype.REVURDERING,
-            organisasjonsnummer: organisasjonsnummer,
-        };
-    });
+export const useMaksdato = (beregningId: string) => {
+    return useSisteUtbetaling(beregningId)?.maksdato;
 };
 
-export const erTidslinjeperiode = (obj: any): obj is Tidslinjeperiode =>
-    (obj as Tidslinjeperiode).beregningId !== undefined;
-
-const isUfullstendig = (vedtaksperiode: Vedtaksperiode) => {
-    return (
-        (vedtaksperiode as UfullstendigVedtaksperiode) !== undefined &&
-        (vedtaksperiode as FullstendigVedtaksperiode) === undefined
-    );
+export const useNettobeløp = (beregningId: string) => {
+    return useSisteUtbetaling(beregningId)?.nettobeløp;
 };
 
-const useHistorikkelement = (beregningId: string) => {
-    const element = usePerson()
-        ?.arbeidsgivere.flatMap((arb: Arbeidsgiver) => arb.utbetalingshistorikk)
-        .find((element: UtbetalingshistorikkElement) => element.id === beregningId);
-    if (!element) throw Error('Fant ikke element i utbetalingshistorikk');
-    return element;
+export const useGjenståendeDager = (beregningId: string) => {
+    return useSisteUtbetaling(beregningId)?.gjenståendeDager;
 };
 
-export const useMaksdato = (beregningId: string) => sisteUtbetaling(useHistorikkelement(beregningId)).maksdato;
-
-export const useNettobeløp = (beregningId: string) => sisteUtbetaling(useHistorikkelement(beregningId)).nettobeløp;
-
-export const useGjenståendeDager = (beregningId: string) =>
-    sisteUtbetaling(useHistorikkelement(beregningId)).gjenståendeDager;
-
 export interface Tidslinjeperiode {
     id: string;
+    unique: string;
```

Comment:
Er `id`-feltet egentlig vedtaksperiodeId? Det ser litt rart ut å ha et id-felt og så et separat felt for å identifisere unikt.